content/docs/user-guide/gems/development/creating/_index.md

```diff
@@ -45,3 +47,14 @@ Your `CMakeLists.txt` file is like any other CMake file. When you create it, kee
   This will be the active target during the invocation of your Gem's `CMakeLists.txt`.
 * You can use the functions available in the core O3DE build system. See the contents of the `cmake` directory in source.
 * Avoid the use of `file(DOWNLOAD ...)`. The package system of O3DE is a robust replacement, and should be used instead.
+
+# Creating an Asset-Only Gem
+
+When you create a Gem without a specified template by using the command `o3de create-gem -gp <path to create a gem at>`, it's created from the "DefaultGem" template. This creates a Gem for packaging both code and assets. You can also create a Gem that provides only assets by specifying the "AssetGem" template through the `create-gem` command:
+
+```cmd
+o3de create-gem --gem-path <path to create gem at> --template-name AssetGem
+# or
+o3de create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem
```

Comment:
Complete the code block with three backquotes.

Reply:
Fixed